client/src/pages/pot-instance/PotInstance.tsx

```diff
@@ -27,18 +31,31 @@ const PotInstance: FC = () => {
     setHotpotStart(false);
   };
 
-  // TODO: should we add time when item was completed to this?
   const addToCookedPot = (foods: PotContent[]) => {
     setCookedPotContent([...cookedPotContent, ...foods]);
+    setFinishedItems([...finishedItems, ...foods]);
+  };
+
+  const removeFromCookedPot = (uniqueid: string) => {
+    const newCookedPotContent = filter(
+      cookedPotContent,
+      (item) => item.id !== uniqueid
+    );
```

Comment:
Doesn't need to be changed, but lodash has a `omitBy` function that lets for make a filtered list like this.

`const newCookedPotContent = _.omitBy(
      cookedPotContent,
      ['id', uniqueid]
    );`